targets/include/ublksrv_tgt.h

```diff
@@ -142,7 +142,7 @@ static inline unsigned short ublk_unique_tag_to_tag(unsigned int unique_tag)
 
 static inline bool ublk_param_is_valid(const struct ublk_params *p)
 {
-	if (p->basic.logical_bs_shift < 9 || p->basic.physical_bs_shift > 12)
```

Comment:
just recently, linux kernel relaxes the logical block size to allow it to be great than 4K, however, ublk driver doesn't support it yet.

However, the above check should have been done on logical block size, instead of physical block size.

Care to fix it?